content-warehouse/src/main/java/contentwarehouse/v1/FetchAcl.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package contentwarehouse.v1;
+
+// [START contentwarehouse_fetch_acl]
+import com.google.cloud.contentwarehouse.v1.DocumentName;
+import com.google.cloud.contentwarehouse.v1.DocumentServiceClient;
+import com.google.cloud.contentwarehouse.v1.DocumentServiceSettings;
+import com.google.cloud.contentwarehouse.v1.FetchAclRequest;
+import com.google.cloud.contentwarehouse.v1.FetchAclResponse;
+import com.google.cloud.contentwarehouse.v1.RequestMetadata;
+import com.google.cloud.contentwarehouse.v1.UserInfo;
+import com.google.cloud.resourcemanager.v3.Project;
+import com.google.cloud.resourcemanager.v3.ProjectName;
+import com.google.cloud.resourcemanager.v3.ProjectsClient;
+import java.io.IOException;
+
+
+public class FetchAcl {
+  public static void fetchAcl() throws IOException {
```

Comment:
```suggestion
  public static void main(String[] args) throws IOException {
```
The format is to use a main method and then call the snippet.